kafka/integration_test.go

```diff
@@ -2007,6 +2007,63 @@ func (its *IntegrationTestSuite) TestAdminConfig() {
 	if topicResult[0].Error.Code() != ErrNoError {
 		t.Fatalf("Failed to delete topic %s: %s", topic, topicResult[0].Error)
 	}
+
+	if !testConsumerGroupProtocolClassic() {
+		// New test case for ResourceType.Group (ResourceGroup)
```

Comment:
Don't add this comment, it's only "new" while it's in this PR. 
Just add a comment as "Test configs that are configured for a specific group."